### PR TITLE
PasscodeActivity changes depending on the theme of the app

### LINKED ISSUE #1902: PassCodeActivity does not change in the dark mode
**Summary:** 

PassCodeActivity does not change in the dark mode and remains the same as in the light mode.

**Steps to reproduce:** 

https://user-images.githubusercontent.com/94394661/221476823-e9ac9e11-1961-4ba9-9aac-e904b204206c.mp4

**Device and Android version:** 

Device : OnePlus Nord
Android 12

### PR DESCRIPTION
**Pull request by Aditya-gupta99**
Fixes #1902 

https://user-images.githubusercontent.com/94394661/221479262-ea9f6cc6-f905-4ca1-9a0b-933fa279c893.mp4

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by Aditya-gupta99** [comment]
@avinash14022002 Now the PR has been separated, please review it.

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/passcode/PassCodeActivity.kt [hunk 1]
```diff
@@ -19,6 +20,7 @@ class PassCodeActivity : MifosPassCodeActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        findViewById<NestedScrollView>(R.id.cl_rootview).setBackgroundColor(android.R.attr.colorBackground)
```
Comment (avinash14022002) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/passcode/PassCodeActivity.kt:23]: @Aditya-gupta99 ,
can you convert it to butterknife?

**Review by avinash14022002 — APPROVED** [review]

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/passcode/PassCodeActivity.kt (hunk 1 above)
Comment (Aditya-gupta99) [reply mifosng-android/src/main/java/com/mifos/mifosxdroid/passcode/PassCodeActivity.kt:23]: Okay @avinash14022002 , I will do that

**Comment by Aditya-gupta99** [comment]
I have opened another PR #1946 .

**Pull request closed without merging**